components/automate-ui/src/app/entities/policies/policy.effects.ts

```diff
@@ -50,12 +50,17 @@ export class PolicyEffects {
   //
   // [1]: https://github.com/ngrx/platform/commit/8d56a6f7e9a0158d30c6a0f335f9805cc7d5a555
 
+  private POLLING_INTERVAL_IN_SECONDS = 300; // 5 minutes
+
+  @Effect()
+  newIamVersionStatus$ = observableInterval(1000 * this.POLLING_INTERVAL_IN_SECONDS)
+    .pipe(mergeMap(this.getIamVersionAction$()));
+
+
   @Effect()
-  getiamMajorVersion$ = this.actions$.pipe(
+  getIamVersion$ = this.actions$.pipe(
     ofType<GetIamVersion>(PolicyActionTypes.GET_IAM_VERSION),
-    mergeMap(() => this.requests.getIamVersion().pipe(
-      map(resp => new GetIamVersionSuccess(resp)),
-      catchError((error: HttpErrorResponse) => observableOf(new GetIamVersionFailure(error))))));
+    mergeMap(this.getIamVersionAction$()));
```

Comment:
Extracted the code to fetch the license version into a function, so that the very same function could be used in `newIamVersionStatus` just above for periodic polling after page load.

Reply:
Good question: It does not require an explicit triggering action. As soon as ngrx-effects initializes it starts up.
Oh, note, however, by "starts up" I do not mean it emits the first request--that's why there exists a separate call in ui.component.ts does. "Starts up" for an `observableInterval` means starts the timer; the first emission is at the end of the interval--you can see a visualization of that here: https://rxviz.com/examples/basic-interval